### PR TITLE
link back to top on Personal page

### LINKED ISSUE #14519: Admin page needs return to top
Admin page needs a link that goes back to the top. We have all the links to the sections on the left (LOVE IT!!! SUPER!!!) but we don't link back to the top of the page.

### PR DESCRIPTION
**Pull request by jancborchardt**
Fix #14519, please review @DeepDiver1975 @owncloud/designers 

Previously there was no sidebar entry for the topmost section in Personal settings, now there is.

### CODE REVIEW

**Comment by karlitschek** [comment]
nice :+1:

File: settings/personal.php [hunk 1]
```diff
@@ -109,7 +109,8 @@
 // add hardcoded forms from the template
 $l = OC_L10N::get('settings');
 $formsAndMore = array();
-$formsAndMore[]= array( 'anchor' => 'passwordform', 'section-name' => $l->t('Personal Info') );
+$formsAndMore[]= array( 'anchor' => 'clientsbox', 'section-name' => $l->t('Sync clients') );
```
Comment (DeepDiver1975) [inline settings/personal.php]: short array for please - thx

**Comment by jancborchardt** [comment]
@DeepDiver1975 done-zo

**Comment by DeepDiver1975** [comment]
:+1:

**Comment by MorrisJobke** [comment]
Works :+1:

**Comment by MorrisJobke** [comment]
Failure unrelated

**Pull request merged**